File: output/CodeLlama/codenet/C++/Java/s628121241.java
import java.util.Scanner;

public class s628121241 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int R = sc.nextInt();
		if (R < 1200)
			System.out.println("ABC");
		else if (R >= 2800)
			System.out.println("AGC");
		else
			System.out.println("ARC");
	}
}

File: output/CodeLlama/codenet/C++/Python/s628121241.py
import math

def gcd(a, b):
    if a == 0:
        return b
    return gcd(b % a, a)

R = int(input())
if R < 1200:
    print("ABC")
elif R >= 2800:
    print("AGC")
else:
    print("ARC")